### ISSUE TITLE
MultiProblem hparams, loss increasing

### ISSUE THREAD

**Issue by tensorator** [tensorflow/tensor2tensor#1565]
### Description

I am trying to use multi-problem. Think of my problem as translating an English layman text to an English domain expert language - like layman text to legal text, or something like that. From a language perspective, both sides of the translation are English with slightly different characteristics. I want to pretrain an unsupervised language model for "legal English" and use the pretrained language model to improve the performance of translator based on Transformer. 

My loss has been diverging for the problem no matter how much I reduce the step size. This model uses transformer_tiny, with mixing_schedule=pretrain, and learning_rate = 0.002.

My problem definition is as follows:

> 
>     @registry.register_problem
>     class LanguagemodelLaymanTranslate(multi_problem.MultiProblem):
>         """Pretrain using keyword data, and then train on translation data QK"""
> 
>         @property
>         def num_generate_tasks(self):
>             return 1
> 
>         def prepare_to_generate(self, data_dir, tmp_dir):
>             pass
> 
>         def __init__(self, was_reversed=False, was_copy=False):
>             super(LanguagemodelLaymanTranslate, self).__init__(was_reversed, was_copy)
> 
>             self.task_list.append(LanguagemodelLegaltextV1())
>             self.task_list.append(TranslateLaymantolegalV1())
> 
>         @property
>         def vocab_type(self):
>             return text_problems.VocabType.SUBWORD
> 

![image](https://user-images.githubusercontent.com/48962493/57001385-44b79500-6b6d-11e9-8ce0-ead3f0dfdb5c.png)

The hparams are as follows:

```

>     @registry.register_hparams
>     def transformer_pretrain_lm_tiny():
>         """Hparams for transformer on LM pretraining on TPU, large model."""
>         hparams = transformer_tiny()
>         hparams.batch_size = 128
>         hparams.multiproblem_mixing_schedule = "pretrain"
>         hparams.learning_rate = 0.001
>         return hparams

```

The component problems are standard text2text problems or text2self problems. I am learning the vocab from language model, and returning it in self.vocab_filename in the translation problem. 

...

### Environment information

```
OS: <your answer here>

$ pip freeze | grep tensor
tensor2tensor==1.12.0
tensorboard==1.12.2
tensorflow-metadata==0.9.0
tensorflow-probability==0.5.0


$ python -V
Python 3.6.6


```

Am I doing it right? I am following the documentation in 

https://github.com/tensorflow/tensor2tensor/blob/master/docs/multi_problem.md

I can provide any information as needed in addition to what's above. Can anyone advise me on how to proceed?
